COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2015 The NixOS and nixpkgs authors
```

Comment:
Please elaborate how nix authors who neither NixOS authors or nixpkgs authors hold copyright on this.